File: BOJ/문제들/b10250.cpp
#include <iostream>
using namespace std;

int main(void) {
  int n;
  cin >> n;
  int a,b,c;
  for(int i=0;i<n;++i) {
    cin >> a >> b >> c;
    // cout << a << b << c << endl;
    int col = c / a;
    int row = c - a * col;
    if(row == 0) {
      col-=1;
      row = a;
    }
    cout << row;
    if(col+1 < 10) {
      cout << '0';
    }
    cout << col+1 << endl;
  }
}

File: BOJ/문제들/b10250.js

// const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');
const input = require('fs').readFileSync('./b10250.txt').toString().split('\n');

let first = false;

const solution = input2 => {
  const input = input2.split(' ');
  let col = parseInt(parseInt(input[2]) / parseInt(input[0]));
  let row = parseInt(input[2]) - col * parseInt(input[0]);
  if(row == 0) {
    row = parseInt(input[0]); 
    col-=1;
  }
  let ans = String(row);
  if(col+1< 10) {
    ans += '0';
  }
  ans += String(col+1);
  console.log(ans);
  return;
}

for(const a of input) {
  if(first == false) {
    first = true;
    continue;
  }

  solution(a);
}

